packages/sanity/src/core/bundles/components/dialog/CreateBundleDialog.tsx

```diff
@@ -31,7 +35,7 @@ export function CreateBundleDialog(props: CreateBundleDialogProps): JSX.Element
 
   const handleOnSubmit = useCallback(
     async (event: FormEvent<HTMLFormElement>) => {
-      if (value.slug) {
+      if (value.slug && versionDocumentExists(bundles, value.slug)) {
```

Comment:
if the versioned item of this document already exists then we shouldn't add it again